app/presenters/sample_transfer_presenter.rb

```diff
@@ -0,0 +1,44 @@
+class SampleTransferPresenter
```

Comment:
This presenter looks nice, but this is introducing a new concept into CDx. I'm wondering if it really improves readability. Most methods could probably be on `FileTransfer` directly, but again the presenter looks nice, so :shrug:

@devduarte Any opinion?

Reply:
I understand this is a new semantical concept. But it's really just a lightweight PORO.

I don't think these methods can go on the model directly. The purpose of this class is to combine model data with contextual information of the current user and institution.

That's a good use case for a presenter. Think of it as a combination of helper methods with context information.
We could do it without, but that would move all that logic into the template or stand-alone helpers. Both cases lead to more boilerplate and less readable code.